src/components/InputTextarea/InputTextarea.story.jsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, text, number } from '@storybook/addon-knobs';
+import { generateVariantsFor } from '../../utils/propsVariants';
+import InputText from './InputTextarea';
+
+const stories = storiesOf('InputTextarea', module);
+stories.add(
+  'InputTextarea',
+  () => {
+    const variants = generateVariantsFor({
```

Comment:
I think you only need to render these for percy, then render just the plain text area with knobs once for the storybook.